docs/spicedb/load_test.md

```diff
@@ -0,0 +1,214 @@
+# SpiceDB Load Testing Guide
+
+<head>
+  <link rel="canonical" href="https://authzed.com/blog/spicedb-load-testing-guide" />
+</head>
```

Comment:
This will get ignored. Think you can specify a canonical url using front matter:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter